project/datasets/modeled/dgen/dataset.json5

```diff
@@ -0,0 +1,110 @@
+{
+  dataset_id: "dgen_DECARB_2023",
+  dataset_type: "modeled",
+  data_source: "dgen",
+  sector_description: "residential and non-residential",
+  description: "dGen Standard Scenarios 2023 with profiles using 2018 as the weather year",
+  origin_creator: "Ashreeta Prasanna",
+  origin_organization: "NREL",
+  origin_contributors: [
+    "Ashreeta Prasanna",
+    "Paritosh Das",
+    "Jane Lockshin",
+  ],
+  origin_project: "Standard Scenarios 2023",
+  origin_date: "2023-11-01",
+  origin_version: "0.1.0",
+  source: "dGen runs for Standard Scenrios 2023",
+  data_classification: "low",
+  tags: [
+    "distributed PV",
+    "conus-2023",
+    "reference",
+    "dsgrid_v0.2.0",
+    "standard_scenarios",
+  ],
+  data_schema_type: "standard",
+  trivial_dimensions: [
+    "weather_year",
+    "scenario",
+  ],
+  data_schema: {
+    load_data_column_dimension: "metric",
+  },
+  dimensions: [
+    { 
+     "class": "County",
+      type: "geography",
+      name: "US_county_FIPS",
+      display_name: "County",
+      file: "dimensions/gen_county_fips_mapping.csv",
```

Comment:
This filename is missing its first letter, i.e., "d".

More importantly, the file is not formatted in a way dsgrid can read--the headings need to be "id" and "name", and you probably need to include the time zone. See, e.g., https://github.com/dsgrid/dsgrid-project-StandardScenarios/blob/main/dsgrid_project/datasets/modeled/resstock/dimensions/conus_2022-resstock_geography_county_fips.csv

Reply:
modified this, but am confused. Is there more concrete guidance on the format? I see some inconsistency in how this is defined for tempo vs resstock/comstock.